skyfield/tests/test_elementslib.py

```diff
@@ -0,0 +1,670 @@
+from skyfield.api import load, Time, tau, load_file
+from skyfield.units import Distance, Angle, Velocity
+from skyfield.constants import DAY_S
+from skyfield.elementslib import OsculatingElements
+from numpy import (array, sin, cos, pi, sqrt, ndarray, float64, repeat, 
+                   seterr, inf, linspace, arccos)
+import os
+
+def ts():
+    yield load.timescale()
+
+def _data_path(filename):
+    return os.path.join(os.path.dirname(__file__), 'data', filename)
+
+ephem = load_file(_data_path('de430-2015-03-02.bsp'))
+jup_ephem = load_file(_data_path('jup310-2015-03-02.bsp'))
+io = jup_ephem['io']
+moon = ephem['moon']
+earth = ephem['earth']
+sun = ephem['sun']
+
+seterr(all='raise')
+
+
+def shift180(num):
```

Comment:
We might want to call this "normalize" or "wrap" rather than "shift" since we're not actually changing the angle's value — though this maneuver is so common and well-understood that I think it already occurs in Skyfield fairly often without being named or made a function call? `norm±pi` would be a great name if only `±` could be used in a function name :)